packages/javascript/bh-shared-ui/src/components/EntityInfo/EntityInfoDataTable.tsx

```
@@ -0,0 +1,128 @@
+            {sections &&
+                sections.map((nestedSection: EntityInfoDataTableProps, nestedIndex: number) => (
+                    <>
+                        <EntityInfoDataTable
+                            {...nestedSection}
+                            additionalSections
+                            data-testid='entity-info-data-table'
+                            key={nestedIndex}
+                            parentLabels={[...(parentLabels as string[]), label]}
+                        />
+                    </>
+                ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)